edb/edgeql/parser/grammar/convert.py

```diff
@@ -0,0 +1,418 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2008-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import parsing
+import pathlib
+from typing import Dict, List, Sequence, Tuple, Callable
+
+
+class EBNF_Item:
+    pass
+
+
+class EBNF_Literal(EBNF_Item):
```

Comment:
Python style is no underscores in class names. I'd probably just drop the entire `EBNF` from the name

Reply:
Moved to separate file. (For now, will move things to tools later)